datameta/views/setpass.py

```diff
@@ -31,29 +31,30 @@
 
 import re
 
-@view_config(route_name='updatepass_api', renderer='json')
-def v_updatepass_api(request):
-    user = security.revalidate_user(request)
-
-    # Try to extract old and new password from request
-    try:
-        old_password = request.json_body["password"]
-        new_password = request.json_body["new_password"]
-    except Error:
-        raise HTTPBadRequest(json_body={'reason' : 'missing field'})
-
-    # Validate old password
-    if not security.check_password(old_password, user.pwhash):
-        raise HTTPUnauthorized()
-
-    # Validate password
-    if len(new_password) < 10:
-        raise HTTPBadRequest(json_body={'reason' : 'password too short'})
-
-    # Set the new password
-    user.pwhash = security.hash_password(new_password)
-
-    return HTTPNoContent()
+# Old API Endpoint
```

Comment:
Please remove the file entirely and also the delete the `add_route` call in `routes.py`

Reply:
Done